CONTRIBUTING.md

```diff
@@ -0,0 +1,183 @@
+# Contributing to Recast and Detour
+
+We'd love for you to contribute to our source code and to make Recast and Detour even better than they are
+today! Here are the guidelines we'd like you to follow:
+
+ - [Code of Conduct](#coc)
+ - [Question or Problem?](#question)
+ - [Issues and Bugs](#issue)
+ - [Feature Requests](#feature)
+ - [Submission Guidelines](#submission-guidelines)
+ - [Git Commit Guidelines](#git-commit-guidelines)
+
+## <a name="coc"></a> Code of Conduct
+This project adheres to the [Open Code of Conduct][code-of-conduct].
+By participating, you are expected to honor this code.
+
+## <a name="question"></a> Got a Question or Problem?
+
+If you have questions about how to use Recast or Detour, please direct these to the [Google Group][groups]
+discussion list. We are also available on [Gitter][gitter].
+
+## <a name="issue"></a> Found an Issue?
+If you find a bug in the source code or a mistake in the documentation, you can help us by
+submitting an issue to our [GitHub Repository][github]. Even better you can submit a Pull Request
+with a fix.
+
+**Please see the Submission Guidelines below**.
+
+## <a name="feature"></a> Want a Feature?
+You can request a new feature by submitting an issue to our [GitHub Repository][github].  If you

```

Comment:
That's what I get for copy-pasting :) Fixed in a new commit. I'll squash all commits once it's ready.